ingestion/src/metadata/ingestion/source/pipeline/dagster/metadata.py

```diff
@@ -192,10 +199,13 @@ def _get_task_status(
                     run.status.lower(), StatusType.Pending.value
                 ),
                 timestamp=Timestamp(
-                    round(convert_timestamp_to_milliseconds(run.endTime))
-                )
-                if run.endTime
-                else None,
+                    round(
+                        convert_timestamp_to_milliseconds(
+                            timestamp=run.endTime
+                            or datetime.datetime.timestamp(datetime.datetime.now())
+                        )
+                    )
+                ),
```

Comment:
@pmbrull the issue here is that when the pipeline is ongoing in dagster, the endTime doesn't show up, in that case, the PipelineStatus won't go in as timestamp is mandatory, None is not allowed.